### DISCUSSION TITLE
hide the main page

### DISCUSSION THREAD

**Discussion by bettykuz** [spencerwooo/onedrive-vercel-index#456] [Q&A] [answered]
Hi guys,

May i know is it possible to hide the main page? i want to put index.htm in main page.

Thanks in advanced.

**Comment by spencerwooo** [comment, marked as answer]
It's a Next.js project, so, although not officially supported, you can always make custom changes.

**Reply by bettykuz** [reply]
thanks, any clue how to achieve that? am not familiar with that

**Reply by myl7** [reply]
You can use [Next.js `basePath`](https://nextjs.org/docs/api-reference/next.config.js/basepath) to serve the app on a subpath like `/share/` and use nginx to reverse-proxy and serve the index page.
It is not easy to describe the detailed steps though there should be many related tutorials on Internet.
